app/alerts.py

```diff
@@ -67,11 +71,20 @@ def build_departments_geojson():
 
 # ----------------------------------------------------------------------------------------------------------------------
 # Sites markers
+
+# We import the cameras's positions from the API that locates the cameras
+
+# Fetching the response in a variable
+response = api_client.get_sites()
+
+# Getting the json data out of the response
+camera_positions = response.json()
+
```

Comment:
I think Flake8 (which verifies whether the code is stylised the right way) has found an unnecessary blank line here. Once you have moved the comment line up as mentioned right below, you can make sure there is only one blank line before the definition of the `build_sites_markers` function and we should be able to pass the styling check!